doc/installation_jetson.md

```diff
@@ -43,8 +43,6 @@ This guide comes with a script that allows to modify the kernel modules with Lib
 
 ### 4. Install with Debian Packages
```

Comment:
Add note regarding the minimum JetPack/CUDA requirements to run precompiled debians

Reply:
Added